docs/evaluation/convolution.rst

```diff
@@ -25,32 +25,175 @@ or the redistribution of the counts as a function of energy
 as modelled by the :term:`RMF` when analyzing astronomical
 X-ray spectra.
 
+.. note::
+
+   In the following documentation we are going to use the term
+   "convolution" to refer to any model that requires the data
+   from another model, such as a model that reprocesses emission
+   into another pass band or that renormalises the data, even if
+   it does not match the meaning of
+   `convolution <https://en.wikipedia.org/wiki/Convolution>`_.
+
+Creating a convolution kernel
+=============================
+
+For convolution-style models, there are normally two steps
+to create a model component:
+
+ 1. create an instance of the convolution kernel
+
+ 2. apply the instance to the model components
+
+The convolution kernel is not an instance of
+:py:class:`sherpa.models.model.ArithmeticModel`, and so
+can not be evaluated by passing it the data grid. Instead,
+the kernel is used to wrap another model expression,
+normally done by passing the model to be wrapped as the
+argument to the model, and the result of this is then
+a model that can be evaluated as normal.
+
+.. note::
+
+   Evaluating the kernel - such as `psf` in the example
+   below - with a grid will raise an error like::
+
+       ModelErr: attempted to create ArithmeticFunctionModel from non-callable object of type ndarray
+
+.. _convolution-psf1d:
+
+One-dimensional convolution with a PSF
+======================================
```

Comment:
What does "one-dimensional convolution with a PSF" even mean? A PSF is usually 2D. I'm trying to come up with a more natural example and if I have an idea I'll comment again. At least, I think this would profit from 1-2 sentences that give an example what this is good for. I think "spectrum", e.g. in the optical: "One example for this is observing a spectrum. The model for the spectrum might have very sharp features, but instrumental effects will broaden those features. Mathematically, we can be expressed as the convolution of the physical spectrum with a broadening function. Note that in X-ray the RMF formalism is used in sherpa, see ...

Reply:
Sherpa has used the term PSF with 1D kernels for longer than you've been using CIAO ;-) You can think of PSF as LSF in this case. I worry that being too explicit about an example - e.g. "Include the grating response in a model" - hides the meaning for people who aren't bothered about spectra.

Reply:
Just to add an example -  1D radial profile analysis can use a 1D PSF generated as a radial profile on the simulated PSF.